lib/go/Makefile

```diff
@@ -46,8 +46,6 @@ PROTOC_URL := https://github.com/protocolbuffers/protobuf/releases/download/v$(P
 PROTOC_TMP_DIR := .protoc
 PROTOC := $(PROTOC_TMP_DIR)/bin/protoc
 
-$(GOBIN)/protoc-gen-go: ../../go.mod
-	go install google.golang.org/protobuf/cmd/protoc-gen-go
```

Comment:
Why not just change this line to
```
go install github.com/golang/protobuf/protoc-gen-go@v1.3.5
```
So we don't need any other changes to the Makefile

Reply:
This did not work for me, it wanted to downgrade other packages in go.mod

Reply:
That's strange. I tried this and it works for me. `github.com/golang/protobuf` stays at v1.5.3 in go.mod

Reply:
Your're right, I was installing `google.golang.org/protobuf/cmd/protoc-gen-go@v1.3.5` and not `github.com/golang/protobuf/protoc-gen-go@v1.3.5`.

I updated the PR, now it's a super simple PR.